modules/ROOT/pages/to-download-an-asset.adoc

```diff
@@ -28,5 +28,9 @@ If you choose As RAML, Exchange downloads the API specification as a zip file co
 * Connector: Downloads as a jar file.
 * Template: Downloads as a jar or zip file.
 * Example: Downloads as a zip file.
+* Policy: Click the down arrow.
++
+If you choose As Mule policy, Exchange downloads the policy definition as a jar file containing a YAML file for the definition. If you choose As Policy definition, Exchange downloads the definition as a YAML file.
```

Comment:
If you choose As Mule policy, Exchange downloads the policy as a jar file containing. If you choose As Policy definition, Exchange downloads the policy definition as a YAML file.

Reply:
What does the jar file contain?

Reply:
Federico explained on Slack that the jar is an executable java package that contains the policy itself, as part of the code that runs, which is always built 1:1 from the source YAML file. (Never JSON.)